index.js

```diff
@@ -75,19 +77,32 @@ const createClient = (profile, userAgent, opt = {}) => {
 			stopovers: false, // fetch & parse previous/next stopovers?
 			// departures at related stations
 			// e.g. those that belong together on the metro map.
-			includeRelatedStations: true
-		}, opt)
+			includeRelatedStations: true,
+		},
+		opt)
```

Comment:
I dislike the style, but for the sake of consistency, can we get all of thee  arguments indented and with a trailing comma?

Reply:
Now it looks like that:

```js
			includeRelatedStations: true,
		}, opt)
```

Do you want a trailing comma after opt?